src/WebUI/UrlSchemaMiddleware.cs

```diff
@@ -36,17 +39,21 @@ public UrlSchemaMiddleware(RequestDelegate next)
             IOptionsSnapshot<CodidactOptions> options)
         {
             // Deal with naughty users that type the IP address directly.
-            if(!request.Host.Host.EndsWith(options.Value.Hostname)) {
+            if (!request.Host.Host.EndsWith(options.Value.Hostname))
+            {
                 return;
             }
 
-            var subdomains = request.Host.Host.TrimSuffix(options.Value.Hostname);
-            var parts = subdomains.Split(".", StringSplitOptions.RemoveEmptyEntries);
+            string subdomains = request.Host.Host.TrimSuffix(options.Value.Hostname);
```

Comment:
[You don't _have_ to use explicit types for variables](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions#implicitly-typed-local-variables), you can use `var` if you want to. This is good either way, whatever you choose.

Reply:
It's an extension method, at first glance it could be unclear that is returned. I'll leave it like that it shouldn't make much of a difference anyway.